metaflow/plugins/metadata/local.py

```diff
@@ -112,6 +115,35 @@ def register_metadata(self, run_id, step_name, task_id, metadata):
         }
         self._save_meta(meta_dir, metadict)
 
+    @classmethod
+    def _mutate_user_tags_for_run(
+        cls, flow_id, run_id, tags_to_add=None, tags_to_remove=None
+    ):
+        # get existing tags
+        run = LocalMetadataProvider.get_object("run", "self", {}, None, flow_id, run_id)
+        if not run:
+            raise MetaflowTaggingError(msg="Run not found (%s, %s)" % (flow_id, run_id))
+        existing_user_tags = run["tags"]
+        existing_system_tags = run["system_tags"]
+
+        for tag in tags_to_remove:
```

Comment:
nit: can also use a set operation here (since you form the sets anyways below)

Reply:
Sounds good.